pkg/nuctl/command/export.go

```diff
@@ -173,7 +187,7 @@ func newExportProjectCommandeer(exportCommandeer *exportCommandeer) *exportProje
 		},
 	}
 
-	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, "Output format - \"yaml\", or \"json\"")
+	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, 'Output format - "json" or "yaml"')
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVarP(&commandeer.output, "output", "o", common.OutputFormatYAML, "Output format - \"json\" or \"yaml\"")
```
This won't compile. Go doesn't support single quotation marks for strings. Please change back to double quotations with escaped quotations within the string.